sdk/communication/azure-communication-chat/swagger/README.md

```diff
@@ -23,21 +23,22 @@ There is one swagger for Chat management APIs.
 
 ```ps
 cd <swagger-folder>
-autorest README.md --java --v4 --use=@autorest/java@4.0.20 --use=@autorest/modelerfour@4.15.442
+autorest README.md --java --v4 --use=@autorest/modelerfour@4.15.442
 ```
 
 ## Update generated files for chat service
 
 To update generated files for chat service, run the following command
 
-> autorest README.md --java --v4 --use=@autorest/java@4.0.20 --use=@autorest/modelerfour@4.15.442
+> autorest README.md --java --v4 --use=@autorest/modelerfour@4.15.442
 
 ### Code generation settings
 
 ```yaml
-tag: package-chat-2023-11-07
+tag: package-chat-2024-03-07
+use: '@autorest/java@4.1.25'
 require:
-    - https://raw.githubusercontent.com/Azure/azure-rest-api-specs/731138d0505c02fcee05d7b5a1417fd10ef5efc3/specification/communication/data-plane/Chat/readme.md
+    - https://raw.githubusercontent.com/Azure/azure-rest-api-specs/1720e5aaa4f5ab094789f72589122baa07133439/specification/communication/data-plane/Chat/readme.md
```

Comment:
This is pointing to the wrong ref. It should be: `https://raw.githubusercontent.com/Azure/azure-rest-api-specs/ac666e993a97fdcda75286b347627c8fd8e7b39d/specification/communication/data-plane/Chat/readme.md`